app/src/components/Button/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyledButton } from './style';
+
+interface Props {
+  onClick: (e?: React.MouseEvent<HTMLButtonElement>) => void;
+  children: React.ReactNode;
+  contained?: boolean;
+  round?: boolean;
+}
+
+const Button = ({
+  onClick,
+  children,
+  contained = true,
+  round = true,
+}: Props) => {
```

Comment:
```suggestion
interface Props {
  onClick: (e?: React.MouseEvent<HTMLButtonElement>) => void;
  children: React.ReactNode;
  size?: 'large'  |  'small';
  width?: number;
  contained?: boolean;
  round?: boolean;
}

const Button = ({
  onClick,
  children,
  size = 'large'
  width = 100
  contained = true,
  round = true,
}: Props) => {
```
- width 값받아서 사용할 수 있게 수정부탁드립니다 width는 숫자로받고 %로 조절할 수 있게 할게요.
- 그리고 height를 조절할 수 있는 size도 추가해주세요. . large는 48px, small 24px 정도
- 적용하시면 스토리북에도 args 수정해서 적용시켜주세요.